flashinfer/moe_ep/kernel_src/cutedsl_megamoe/shim/correctness.py

```
@@ -173,41 +135,21 @@ def main(argv: list[str] | None = None) -> int:
         default=1e-2,
         help="Relative tolerance for reference compare (default: 1e-2).",
     )
-    args = parser.parse_args(argv)
+    parser.parse_args(argv)
 
     _, rank, world_size, _ = bootstrap_dist()
```

Ruff (0.15.21)

[warning] Unpacked variable `rank` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)